webserver/Models/Controllers/RealEstatesController.cs

```diff
@@ -0,0 +1,154 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using webserver.Data;
+using webserver.Models;
+using webserver.Models.DTOs;
+
+namespace webserver.Controllers;
+
+[ApiController]
+[Route("api/v1/realestates")]
+public class RealEstatesController : ControllerBase {
+    
+    private readonly WebserverContext _context;
+
+    public RealEstatesController(WebserverContext context) {
+        _context = context;
+    }
+
+    [HttpGet("unique/{id}")]
+    public IActionResult ReadRealEstate(int id){
+
+        var realEstate = _context.RealEstates.Find(id);
+        if(realEstate==null){
+            return NotFound();
+        }
+
+        var response = new {
+            data = new {
+                type = "RealEstate",
+                attribute = new RealEstateDTO (id, realEstate.Name, realEstate.Address, realEstate.Price, realEstate.CompanyId)
+            }
+        };
```

Comment:
I'd like to know if there is an easy way to answer JSON API.
I think there is, but this way works.

Reply:
JsonConvert.SerializeObject()

Thanks for the heads-up